content/pt-br/docs/concepts/architecture/nodes.md

```diff
@@ -289,11 +287,7 @@ Message:        Pod was terminated in response to imminent node shutdown.
 
 {{< feature-state state="alpha" for_k8s_version="v1.23" >}}
 
-Assuming the following custom pod
-[priority classes](/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass)
-in a cluster,
-
-Para fornecer mais flexibilidade durante o desligamento gradual do nó em torno da ordem de pods durante o desligamento, o desligamento gradual do nó respeita a PriorityClass for Pods, desde que você tenha ativado esse recurso em seu cluster. O recurso permite que o cluster defina explicitamente a ordem dos pods durante o desligamento gradual do nó com base em [classes de prioridade](/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass).
+Assumindo as seguintes [classes de prioridade](/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass) do pod em um cluster, para fornecer mais flexibilidade durante o desligamento gradual do nó em torno da ordem de pods durante o desligamento, o desligamento gradual do nó respeita a PriorityClass dos Pods, desde que você tenha ativado esse recurso em seu cluster. O recurso permite que o cluster defina explicitamente a ordem dos pods durante o desligamento gradual do nó com base em [classes de prioridade](/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass).
```

Comment:
The first sentence ("assumindo as seguintes classes de prioridade do pod em um cluster") does not exist in the original documentation. I believe this is a typo.